cmd/gcp-controller-manager/node_annotator.go

```diff
@@ -334,6 +444,50 @@ func parseLabels(labelString string) (map[string]string, error) {
 	return labels, nil
 }
 
+func parseTaints(taintsString string) ([]core.Taint, error) {
```

Comment:
Wouldn't this validation logic already exist in K8s and be re-usable?

Reply:
Done.